perf.cpp

```diff
@@ -32,9 +33,21 @@ void Perf::StopTimer(std::string phase_name) {
   }
 }
 
+void Perf::CalculateFps() {
+  float average_fps =
+      kMsPerSecond / (phase_accumulated_time_["game_loop"] /
+                      static_cast<float>(print_freq_in_frames_));
+  std::cout << "FPS: " << average_fps << std::endl;
+}
+
 void Perf::ReportResults() {
   total_frames_++;
-  if (enabled_ && (total_frames_ % print_freq_in_frames_ == 0)) {
+  if (enabled_ && (total_frames_ % print_freq_in_frames_ == 0) &&
+      total_frames_ != 0) {
+    if (phase_accumulated_time_.find("game_loop") !=
+        phase_accumulated_time_.end()) {
+      CalculateFps();
```

Comment:
It would be nice if the FPS was part of the "Performance Report". How bout, make CalculateFps return the fps rather than print it to the screen.

Then you can do this:

```
    std::cout << "Performance Report After " << print_freq_in_frames_
              << " Frames" << std::endl;
    std::cout << "Average FPS: " << CalculateFps() << std::endl;
    for (const auto& [name, time] : phase_accumulated_time_) {
```